tools/system_calls.py

```diff
@@ -5,9 +5,19 @@
 import pyscreenshot as ImageGrab
 from subprocess import call
 import webbrowser
+from Xlib.error import DisplayNameError
 
 from tools.common import UPLOAD_DIRECTORY
 
+try:
+    from pynput.keyboard import Controller
+    x_display = True
+except DisplayNameError:
+    print("Couldn't find connected DISPLAY. Keyboard input is disabled.")
+    x_display = False
+
+urls = []
+urls.extend([None]*10)
```

Comment:
Creating empty list with length of 10 Nones is not the best idea.
It’s be better to use queue and pop elementy from the end of list and fill list dynamically.